ansible/roles/schulcloud-server-core/templates/amqp-files-deployment.yml.j2

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: amqp-files-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: amqp-files
+spec:
+  replicas: {{ API_FILE_STORAGE_REPLICAS|default("1", true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      #maxUnavailable: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: amqp-files
+  template:
+    metadata:
+      labels:
+        app: amqp-files
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+      - name: amqp-files
+        image: {{ SCHULCLOUD_SERVER_IMAGE }}:{{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        imagePullPolicy: IfNotPresent
+        envFrom:
+        - configMapRef:
+            name: api-configmap
+        - secretRef:
+            name: api-secret
+        command: ['npm', 'run', 'nest:start:files-storage-amqp:prod']
+        resources:
+          limits:
+            cpu: {{ API_FILE_STORAGE_CPU_LIMITS|default("2000m", true) }}
```

Comment:
I am not sure ...but if we seperate this in additonal app it make sense that we can managed cpu and memory different to the fles storage.